### ISSUE TITLE
Container change

### ISSUE THREAD

**Issue by mnovosel2** [luruke/barba.js#14]
Hello,
I was not able to find how to change .barba-container and .barba-wrapper container classes into something custom.

**Comment by luruke** [comment]
Hello @mnovosel2
Thanks for raising this up.
Is something I wrote on the documentation but I didn't had time to make it right.

At the moment the best way will be to replace at runtime the functions `Barba.Pjax.getNamespace` and `Barba.Pjax.putContainer`.

In the next release I will try to make it more easily editable with some variable.

Let me know if you have problems ;)

**Comment by mnovosel2** [comment]
Can you please give me code example. I'm not really sure in which order I have to call these functions. Thank you

**Comment by luruke** [comment]
Hi @mnovosel2 !
I just published a new version (`0.0.8`);

Now, before starting Barba with `Barba.Pjax.start()` you can modify those static values:

```javascript
Barba.Pjax.Dom.dataNamespace // default as 'namespace'
Barba.Pjax.Dom.wrapperId // default as 'barba-wrapper'
Barba.Pjax.Dom.containerClass // default as 'barba-container'
```

Let me know if it works for you or if you encounter in any problem.
